### PR TITLE
Byi setup script

### PR DESCRIPTION
**Pull request by bin3377**
add a bash script for creating source deploy fluent
with following change:
 - remove prometheus-operator and cAdvisor since we no longer have metrics with these jobs
 - document update for process/script usage/relabel/patch Prometheus service, etc.

FYI - @lei-sumo @ggarg2906sumo

### CODE REVIEW

File: deploy/kubernetes/setup.sh [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  echo "Namespace 'sumologic' exists, skip creating."
+fi
+
+set +e
+echo "Creating secret 'sumologic'..."
+kubectl -n sumologic describe secret sumologic &>/dev/null
+retVal=$?
+set -e
+if [ $retVal -eq 0 ]; then
+  echo "Deleteing old secret 'sumologic'..."
```
Comment (samjsong) [inline deploy/kubernetes/setup.sh]: typo: `Deleting`

File: deploy/README.md [hunk 2]
```diff
@@ -11,24 +11,46 @@ The diagram below illustrates the components of the Kubernetes metric collection
...
-## Step 1: Create hosted collector and HTTP sources 
-In this step you create a Sumo Logic hosted collector with a set of HTTP sources to receive your Kubernetes metrics. 
+## Step 1: Create Sumo collector and deploy Fluentd
 
-**To create a hosted collector and an HTTP source**
+In this step you create a Sumo Logic hosted collector with a set of HTTP sources to receive your Kubernetes metrics; creates Kubernetes secrets for the HTTP sources created; and deploy Fluentd using a Sumo-provided .yaml manifest.
+
+### Automatic with setup script
+
+This approach requires the access to Sumo Logic API.
```
Comment (frankreno) [inline deploy/README.md]: let's specify Collector API

File: deploy/README.md [hunk 3]
```diff
@@ -121,7 +141,8 @@ Install `prometheus-operator` using Helm:
 ```sh
 helm repo update \
    && helm install stable/prometheus-operator --name prometheus-operator --namespace sumologic -f overrides.yaml
-   ```
+```
+
 __NOTE__ If credentials were created earlier, add `--no-crd-hook` to the end of the command.
```
Comment (frankreno) [inline deploy/README.md]: Change credentials to Custom Resource Definitions - that is what CRD

File: deploy/README.md [hunk 4]
```diff
@@ -121,84 +142,92 @@ Install `prometheus-operator` using Helm:
...
+<filter prometheus.datapoint**>
+  @type prometheus_format
+  relabel container_name:container,pod_name:pod
   inclusions { "namespace" : "kube-system" }
 </filter>
 ```
 
 This:
 
 * Trims the service metadata from the metric datapoint.
```
Comment (samjsong) [inline deploy/README.md]: I think the above example no longer trims `service` metadata

**Pull request merged**